### PR TITLE
[localization] Fix indentation in class template synopses.

### LINKED ISSUE #1205: Synopses in [localization] use non-standard indentation style
A lot of the synopses in Clause 22 don't indent the second line of a class template, i.e. the say

    template <typename T>
    class foo {
    public:

instead of our usual:

    template <typename T>
      class foo {
      public:

This might be worth cleaning up. While you're there, there's also ample opportunity to look out for other idiosyncratic spacing and indentation.

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #1205.

### CODE REVIEW

File: source/locales.tex [hunk 1]
```diff
@@ -1947,54 +1948,55 @@
 \indexlibrary{\idxcode{ctype<char>}}%
 \begin{codeblock}
 namespace std {
-  template <> class ctype<char>
-    : public locale::facet, public ctype_base {
-  public:
-    using char_type = char;
+  template <>
+    class ctype<char>
+      : public locale::facet, public ctype_base {
```
Comment (tkoeppe) [inline source/locales.tex]: Bring the base class up on the same line like everywhere else while you're at it.

File: source/locales.tex [hunk 2]
```diff
@@ -3194,48 +3196,48 @@
...
+      using char_type = charT;
+      using iter_type = OutputIterator;
+
+      explicit num_put(size_t refs = 0);
+
+      iter_type put(iter_type s, ios_base& f, char_type fill, bool v) const;
+      iter_type put(iter_type s, ios_base& f, char_type fill, long v) const;
+      iter_type put(iter_type s, ios_base& f, char_type fill, long long v) const;
+      iter_type put(iter_type s, ios_base& f, char_type fill,
+                    unsigned long v) const;
```
Comment (tkoeppe) [inline source/locales.tex]: I think we don't need these next couple of linebreaks. If you remove them now, I will never have to go back to Clause 22 again, which I would appreciate.

**Comment by tkoeppe** [comment]
Thanks a lot!

**Comment by jensmaurer** [comment]
Fixed as desired.

**Pull request merged**